signature/spec/policy.go

```diff
@@ -0,0 +1,127 @@
+type Policy struct {
```

Comment:
This should be covered in #50 . The internal evaluation implementation from `signature/policy_types.go` and the comments in there have a different focus from end-user documentation.